bash_completion

```diff
@@ -1878,7 +1878,9 @@ _longopt()
 
     if [[ "$cur" == -* ]]; then
         COMPREPLY=( $( compgen -W "$( LC_ALL=C $1 --help 2>&1 | \
-            command sed -ne 's/.*\(--[-A-Za-z0-9]\{1,\}=\{0,1\}\).*/\1/p' | sort -u )" \
+            command grep --basic-regexp --line-number --only-matching -- '--[-A-Za-z0-9]\{1,\}=\{0,1\}' | \
+            mawk -F: '{ if (!options[$1]) options[$1] = $2 } END { for (line in options) printf("%s\n", options[line]);  }' | \
```

Comment:
`mawk` isn't portable either, plain awk would be acceptable but I hope these commands can be simplified overall.

Reply:
Right. Clear bash is sufficient.